package.json

```diff
@@ -83,6 +83,7 @@
     "babel-runtime": "6.26.0",
     "classnames": "2.2.5",
     "form-serialize": "0.7.2",
+    "react-motion": "^0.5.2",
```

Comment:
This is added to the wrong place. It should be added to `react-ui-core`. The root does not get exported.